src/redux/reservation/thunk.js

```diff
@@ -0,0 +1,37 @@
+// reservationThunks.js
+
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import axios from 'axios';
+
+const url = 'https://bookitnow-kk0q.onrender.com/api/v1'; // Replace with your API endpoint
+
+export const fetchReservations = createAsyncThunk('reservations/fetchReservations', async (userId, thunkAPI) => {
+  try {
+    const response = await axios.get(`${url}/users/${userId}/reservations`);
+    //    console.log('Redux Store State:', response.data);
```

Comment:
- [ ] Kindly, remove this piece of old commented code.  that way, the repo's cleanliness will be improved. 👍